composer.json

```diff
@@ -29,6 +29,7 @@
   "require-dev": {
     "scriptfusion/phpunit-immediate-exception-printer": "^1",
     "phpunit/phpunit": "^5.7",
+    "silverstripe/recipe-cms": "~1",
```

Comment:
It's better to require the CMS module than the recipe for it

Reply:
This is specifically for `require-dev`, basically, get the `Page` class stuff installed in Travis